rfcs/convergence/authoring-stories.md

```diff
@@ -267,13 +271,73 @@ TBA
 
 TBA
 
-### 7. how structure stories in the storybook nav tree
+### 7. dissecting big story files into smaller ones
 
-TBA
+Components with complex API require many long stories, both for documentation and for e2e testing. This creates big story files, which are hard to maintain. How can these files be dissected into smaller ones?
+
+**Proposal:**
 
-### 8. dissecting big story files into smaller ones
+1. Every component can have at most one `.stories.tsx` file with default export which configures metadata about the component. This file must be called `Component.stories.tsx`, for example `Button.stories.tsx`.
+2. If putting all stories into one file would make it too long, individual stories might be put into additional `.stories.tsx` files as a named export, and then re-exported from `Component.stories.tsx` file like this: `export * from ‘./IndividualStoryFile.stories’;`
+3. If individual story files are employed, `Component.stories.tsx` file must not contain any stories.
 
-TBA
+### 8. location and naming convention
+
+1. Story files should be in the same folder as the component which they are meant to show off. This will usually be the component which is the most unique to the story.
+2. Name of every .stories.tsx file should start with name of a component which it is targeting, to improve colocation in alphabetical file ordering. If appropriate, story name should be adjusted via configuration to a well readable name, suitable for documentation, like this:
+
+```tsx
+export const ButtonPrimary = (props: ButtonProps) => <Button {...props}>Text</Button>;
+ButtonPrimary.args = {
+  primary: true,
+};
+ButtonPrimary.storyName = 'Better story name';
+```
+
+### 9. UX of stories
+
+#### first story is called default
```

Comment:
```suggestion
#### main component story

- is named `Default` -> `export const Default = () => {...}`
```

Reply:
What do you think about "default story"? I think it’s better to keep naming consistent

Reply:
TBH not sure :D , after our discussion yesterday it would make sense to name it `Playground` or `ApiPlayground`  which will express the intent in better way than `Default`